dynamiqs/utils/wigner.py

```diff
@@ -19,16 +19,18 @@ def wigner(
     n_pixels: int = 200,
     method: Literal['clenshaw', 'fft'] = 'clenshaw',
 ) -> tuple[Tensor, Tensor, Tensor]:
-    """Compute the wigner distribution of a state vector or density matrix.
+    """Compute the wigner distribution of a ket or density matrix.
 
     Args:
-        state: State vector or density matrix.
-        x_max: Maximum value of x for which to compute the wigner distribution.
-        p_max: Maximum value of p for which to compute the wigner distribution.
+        state (..., n, 1) or (..., n, n): State vector or density matrix.
+        x_max (float): Maximum value of x for which to compute the wigner distribution.
+            Defaults to `2 * pi`.
```

Comment:
```suggestion
        x_max: Maximum value of x.
```
- The default appears in the function signature, no need to say it except if it has a strange value/behaviour.
- The type appears in the function signature, we just use it for tensors to specify their dimension.

Reply:
This applies to the whole PR, we can remove the type between parentheses in the docstring if it' no different from the function signature, and we can remove the indication of default values.